internal/backend/storage.go

```diff
@@ -33,6 +34,7 @@ type Storage interface {
 	GetCRL() (string, error)
 	WriteCAChain(pemBundle []string) error
 	GetCAChain() ([]string, error)
+	ListExpiredCertificates(int) ([]*big.Int, error)
```

Comment:
Is this what is replacing `/pki/crl/purge`?

Reply:
Kind of

Reply:
No, it literally exists as a method by which to find expired certs (used when creating a CRL or purging the certificate store since by default certificates will exist until they are purged).